test/baseline/opt-hashes-neg.test

```diff
@@ -0,0 +1,23 @@
+2008/01/01 January
+    Expenses:Books                            $10.00
+    Assets:Cash
+
+2008/01/31 End of January
+    Expenses:Books                            $10.00
+    Assets:Cash
+
+2008/02/01 February
+    ; Hash: f80797304b8d9acb2084772364ee717f0b724cddb1a9ccb6a3fb8d9cb803fe3b9b7a458211a7fe818578b6ec75a1a31acbcf8e2926ea349141568252966189aa
```

Comment:
Hashes are rather long, eg filling one and a half lines of an 80-char terminal in print output. Could you get similar benefit with a shorter one ?

Reply:
It will only require a comparison match up to the length given, so you can enter the first half of the hash, for example, and it will still give you a huge amount of assurance.

Reply:
Nixpkgs uses a shorter hashes, e.g. `hash = "sha512-IakZy6XsAE39awjzQI+R11JCPeQSaibx6+uX8Iea5WdlCundeovnPwSAi6RzzZl9dr2UftzzEiF4Aun8VMtqVA==";` (see [`musikcube/default.nix:41`](https://github.com/NixOS/nixpkgs/blob/24682ced2e23fff4144b790c3a0d18382b713a8c/pkgs/applications/audio/musikcube/default.nix#L41)), yet I do not know how these are shortened/encoded.

Nix offers [libutil](https://github.com/NixOS/nix/tree/master/src/libutil) although I'm uncertain how mature the library is on terms of use from third-party software and how sensible and feasible it is to integrate it into ledger.